src/concrete/ml/sklearn/base.py

```diff
@@ -2093,11 +2093,11 @@ def predict(self, X: Data, fhe: Union[FheMode, str] = FheMode.DISABLE) -> numpy.
 
         topk_labels = []
         for query in X:
```

Comment:
by the way @kcelia I have a question on this : why can't we use `.predict(self, X, fhe)`here ? Because here we basically loop and run `predict` on queries with shape (1, n) but we handle (m,n) shapes ! I tried to change it to make this simpler but found some issues when running with `fhe="disable`, does that mean the `_inference` method was only made for (1, n) samples ?

Reply:
The sort algorithm is designed to sort an array of 1d.
So, you will have to loop at some point.

Reply:
ok got it, yeah it's too bad that we therefore need to have somewhat 2 unnecessary loops here instead of 1 but oh well, thanks !